spring-cloud-aws-autoconfigure/src/test/java/org/springframework/cloud/aws/autoconfigure/appconfig/AwsAppConfigBootstrapConfigurationTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2013-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.aws.autoconfigure.appconfig;
+
+import java.lang.reflect.Method;
+
+import com.amazonaws.AmazonWebServiceClient;
+import com.amazonaws.services.appconfig.AmazonAppConfigClient;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import org.springframework.util.ReflectionUtils;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AwsAppConfigBootstrapConfigurationTest {
```

Comment:
Can we move the tests to use `ApplicationContextRunner`?

Reply:
Yep. No problem.